azure-maven-plugin-lib/src/main/resources/MessageTemplates-aca.yaml

```diff
@@ -0,0 +1,169 @@
+  select_none:    "You have not selected any Azure Container App, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-registry
+promote:
+  header:       "Available Azure Container Registries:"
+  many:         "Select Azure Container Registry for deployment:"
+message:
+  empty_options:  "There are no Azure Container Registry in current subscription, please create it."
+  auto_select:    "There is only one Azure Container Registry (***${ registries[0].name() }***) in current subscription, will use it automatically."
+  select_none:    "You have not selected any Azure Container Registry, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               select-identity
+promote:
+  header:       "Available User Assigned Identities:"
+  many:         "Select User Assigned Identity for deployment:"
+message:
+  empty_options:  "There are no User Assigned Identity  in current subscription, please create it."
+  auto_select:    "There is only one User Assigned Identity (***${ identities[0].name() }***) in current subscription, will use it automatically."
+  select_none:    "You have not selected any User Assigned Identity, 'config' will terminate."
+required:         true
+auto_select:      true
+default_index:    0
+
+---
+id:               configure-app-name
+promote:          "Input the app name (***${default_val}***):"
+resource:         App
+property:         appName
+default_val:          '${project.getArtifactId().replaceAll("[^-a-z0-9]", "").toLowerCase()}'
+expression:       true
+required:         true
+
+---
+id:               configure-deployment-type
+promote:          "Input deployment type for apps (***${schema.default}***):"
+resource:         App
+property:         deploymentType
+
+---
+id:               configure-code-dir
+promote:          "Input directory for source code (***${schema.default}***):"
+resource:         App
+property:         directory
+
+---
+id:               configure-artifact-dir
+promote:          "Input directory for artifact (***${schema.default}***):"
+resource:         App
+property:         directory
+
+---
+id:               configure-image
+promote:          "Input image (***${schema.default}***):"
+resource:         App
+property:         image
+
+---
+id:               configure-registry-name
+promote:          "Input registry name:"
+resource:         App
+property:         registryName
+
+---
+id:               configure-min-replicas
+promote:          "Input min replicas for apps [***${schema.minimum}***-***${schema.maximum}***] (***${schema.default}***):"
+resource:         App
+property:         minReplicas
+
+---
+id:               configure-max-replicas
+promote:          "Input max replicas for apps of standard consumption plan [***${schema.minimum}***-***${schema.maximum}***] (***${schema.default}***):"
```

Comment:
Max replica has default value to `10`, which is likely to be too large. I suggest we start with 2, so the default scale range is 0 to 2.

Reply:
The ACA quickstart app use 0-10 as the default min/max replicas. Keep the same logic.